include/nbl/video/IPhysicalDevice.h

```diff
@@ -148,49 +328,80 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
             // Whether `shaderSubgroupQuad` flag refer to all stages where subgroup ops are reported to be supported.
             // See SLimit::subgroupOpsShaderStages.
             bool shaderSubgroupQuadAllStages = false;
-            bool drawIndirectCount = false;
-            bool multiDrawIndirect = false;
-            bool samplerAnisotropy = false;
-            bool geometryShader    = false;
 
-            // RayQuery
+            //--> VkPhysicalDeviceRayQueryFeaturesKHR
             bool rayQuery = false;
 
-            // AccelerationStructure
+            //--> VkPhysicalDeviceAccelerationStructureFeaturesKHR
             bool accelerationStructure = false;
             bool accelerationStructureCaptureReplay = false;
             bool accelerationStructureIndirectBuild = false;
             bool accelerationStructureHostCommands = false;
             bool descriptorBindingAccelerationStructureUpdateAfterBind = false;
 
-            // RayTracingPipeline
+            //--> VkPhysicalDeviceRayTracingPipelineFeaturesKHR
             bool rayTracingPipeline = false;
             bool rayTracingPipelineShaderGroupHandleCaptureReplay = false;
             bool rayTracingPipelineShaderGroupHandleCaptureReplayMixed = false;
             bool rayTracingPipelineTraceRaysIndirect = false;
             bool rayTraversalPrimitiveCulling = false;
 
-            // Fragment Shader Interlock
+            //--> VkPhysicalDeviceFragmentShaderInterlockFeaturesEXT
             bool fragmentShaderSampleInterlock = false;
             bool fragmentShaderPixelInterlock = false;
             bool fragmentShaderShadingRateInterlock = false;
 
-            // Queries
-            bool allowCommandBufferQueryCopies = false;
-            bool inheritedQueries = false;
 
-            // Buffer Device Address
+            //--> VkPhysicalDeviceBufferDeviceAddressFeaturesKHR
             bool bufferDeviceAddress = false;
+            //VkBool32           bufferDeviceAddressCaptureReplay;
+            //VkBool32           bufferDeviceAddressMultiDevice;
+            
+            //--> Nabla:
+            bool dispatchBase = false;
+            bool drawIndirectCount = false;
```

Comment:
pretty sure that's a feature in Vulkan

Reply:
probably has a different name

Reply:
@devshgraphicsprogramming 

``dispatchBase`` exists on vulkan with no need for feature checking and is not valid to use in gl because we don't implement it.

https://github.com/Devsh-Graphics-Programming/Nabla/blob/fddcc6b81cfcb4229648043539c41ba3c2ba095c/src/nbl/video/COpenGLCommandBuffer.cpp#L833-L835

https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkCmdDispatchBase.html

Reply:
About `drawIndirectCount` I have no idea why it is a bool and in features but It's exposed under `maxDrawIndirectCount` in limits

Reply:
> @devshgraphicsprogramming
> 
> `dispatchBase` exists on vulkan with no need for feature checking and is not valid to use in gl because we don't implement it.
> 
> https://github.com/Devsh-Graphics-Programming/Nabla/blob/fddcc6b81cfcb4229648043539c41ba3c2ba095c/src/nbl/video/COpenGLCommandBuffer.cpp#L833-L835
> 
> https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkCmdDispatchBase.html

yes make it fail in GL